src/interfaces/store-interfaces/state-interfaces/businessInformation-interface.ts

```diff
@@ -5,6 +5,6 @@ export interface BusinessInformationIF {
   legalType: EntityTypes
   identifier: string
   legalName?: string
-  foundingDateTime?: string
+  foundingDate?: string // actually date-time
```

Comment:
Argh, inconsistent naming from API.
I added the comment to clarify that this isn't an absolute date (eg, "2021-03-18") but is instead a UTC date-time (eg, "2021-03/18T17:38:00Z").

If anyone has a good naming scheme to distinguish the following different types, let me know!
- absolute date (a day in Pacific tz)
- a date-time (in UTC)
- a Date object